docs_guide/sphinx_guide/how_to_add_sections.rst

```diff
@@ -0,0 +1,95 @@
+####################################
+How to add sections to documentation
+####################################
+
+.. include:: ../others/how-to_type.rst
+
+This guide shows how to separate your documents in parts. 
+
+.. include:: ../others/general_pre-requisites.rst
+
+You can organize your documents into:
+
+* Sections.
+* Subsections.
+* Subsubsections.
+* Paragraphs.
+
+Create sections
+===============
+
+Section headers are created by underlining (and optionally overlining) the section title with a punctuation character, at least as long as the text:
```

Comment:
When I wrote that initally I tried to make people use some specific punctuation characters (`=` for section, `^` for subsection and `"` for paragraph) to keep consitency. Not sure if it's a good idea to remove that.